.env.example

```diff
@@ -1 +1,9 @@
-BROWSER="none"
\ No newline at end of file
+BROWSER="none"
```

Comment:
Looking generally ok, but to full review you have to fix an error that I get on PR deploy (unathorized domain) while trying to log in :P

Reply:
Unfortunately Firebase does not support wildcard domains. There aren't any hacks around this that I know of.

Reply:
that sucks, at least then you have to add the domain on which main is deployed

Reply:
that was already added

Reply:
Pls, can you add the main deploy link in readme here? :D